metricbeat/module/windows/services/_meta/fields.yml

```diff
@@ -0,0 +1,36 @@
+- name: services
+  type: group
+  description: >
+    `services` contains the status for windows services.
+  fields:
+    - name: uptime
```

Comment:
This can be simplified if you use a dotted key name. The tooling with expand it for the purposes of the index template.

```
  fields:
    - name: uptime.ms
      type: long
      format: duration
      input_format: milliseconds
      description: >
        The service uptime in milliseconds.

    - name: service_name
```